src/submission/exceptions/exceptions.ts

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
applying exception logic moved out of main `submission-service` file and into this file
returns the schema errors filtered by whether there is an exception or not
and the record with normalized exceptions value if applicable